content/v2.0/reference/api/client-libraries/js.md

```diff
@@ -1,12 +1,167 @@
 ---
-title: InfluxDB JavaScript client library
+title: JavaScript client library
 list_title: JavaScript
-description: Use the JavaScript client library to interact with InfluxDB.
-external_url: https://github.com/influxdata/influxdb-client-js
+description: >
+  Use the JavaScript client library to interact with InfluxDB.
 menu:
   v2_0_ref:
     name: JavaScript
     parent: Client libraries
-    url: https://github.com/influxdata/influxdb-client-js
+v2.0/tags: [client libraries, JavaScript]
+aliases:
+  - /v2.0/reference/api/client-libraries/js-cl-guide/
 weight: 201
 ---
+
+Use the [InfluxDB JavaScript client library](https://github.com/influxdata/influxdb-client-js) to integrate InfluxDB into JavaScript scripts and applications.This client supports both browser environments and NodeJS environments. 
+
+This guide presumes some familiarity with JavaScript, browser environments, and InfluxDB.
+If just getting started, see [Getting started with InfluxDB](/v2.0/get-started/).
+
+## Before you begin
+
+1. Install NodeJS:
+
+    ```sh
+    brew install node 
+    ``` 
```

Comment:
I don't think we should assume macOS here or even show them how to install NodeJS. Just link `Install NodeJS` to NodeJS' installation instructions. Looks like there are a few options:

- https://nodejs.org/en/download/package-manager/
- https://nodejs.org/en/download/